client/reader/route/index.js

```diff
@@ -1,33 +1,12 @@
-import config from '@automattic/calypso-config';
-
 const FEED_URL_BASE = '/read/feeds/';
 const SITE_URL_BASE = '/read/blogs/';
 
-const DISCOVER_SITE_ID = config( 'discover_blog_id' );
```

Comment:
The `discover_blog_id` config var doesn't seem to be used anymore. Perhaps we can clean it up as well?

Reply:
I started removing these but noticed it's still in the Editing Toolkit config:

`apps/editing-toolkit/editing-toolkit-plugin/help-center/src/config.js`

Any ideas why that might be? Is it used by the mobile apps?

Reply:
Ah - I've got it. I think the Reader full post component is used by the Help Center, and we used to do the `isDiscoverPost` check there. It's no longer needed, so I'll remove from all the config files. 👍

Reply:
Removed ✅